core/src/test/scala/integration/kafka/server/QuorumTestHarness.scala

```diff
@@ -444,7 +444,6 @@ object QuorumTestHarness {
 
   // The following parameter groups are to *temporarily* avoid bugs with the CONSUMER group protocol Consumer
   // implementation that would otherwise cause tests to fail.
-  def getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17960: stream.Stream[Arguments] = getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly
```

Comment:
there is another test still disabled using this https://github.com/apache/kafka/blob/9afb27481887e001a1876b019c325c778cbd5676/core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala#L2170, I believe it was passing locally so let's just double check and maybe we can enable it too? (or leave this)

Reply:
Good catch!

Reply:
I changed the other test too. It passes a few times locally.